swan-lake/learn/security/writing-secure-ballerina-code.md

```diff
@@ -113,50 +116,3 @@ function sanitizeSortColumn (string columnName) returns @untainted string {
 // ...
 ```
 
-## Securing Passwords and Secrets
```

Comment:
Will need to re write after fixing https://github.com/ballerina-platform/ballerina-lang/issues/28067